### ISSUE TITLE
Applications vs. Bookmarks

### ISSUE THREAD

**Issue by gkovacsp** [pawelmalak/flame#197]
Besides having a bigger icon/text, what is the intended difference between an Application and a Bookmark?

**Comment by pawelmalak** [comment]
Apps section is intended for local/self-hosted applications and services, while Bookmarks are for external links like social media, news, blogs etc. Of course, nothing stops you from adding an application as a bookmark and vice versa but that was the original idea to have them separated.

Also, Flame was created as an interactive version of [SUI](https://github.com/jeroenpardon/sui) so main layout was inherited from there.

**Comment by gkovacsp** [comment]
Yes, that's what I figured as well. I've also checked SUI and was wondering why the URL of the application is displayed? What exra info does it contain?

I'm not sure if you know Heimdall, their bookmarks are able to get extra info out from the linked apps (e.g.: Portainer: running containers). That could make the applications more special.

**Comment by gkovacsp** [comment]
Here it is: https://github.com/linuxserver/Heimdall

**Comment by gkovacsp** [comment]
![image](https://user-images.githubusercontent.com/1654787/142223388-ea9a13b6-470e-4fe8-85d3-5816c1cf557e.png)

**Comment by gkovacsp** [comment]
This is my custom CSS workaround to remove the URL:

```
.AppCard_AppCardDetails__tbAhY span {
    font-size: 0px !important;
    visibility: hidden;
}
```
